pkgs/development/python-modules/pydeck/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonApplication, fetchPypi, pkgs }:
+
+buildPythonApplication rec {
```

Comment:
But I don't think you actually want to be using `buildPythonApplication` at all - `pydeck` looks like an importable module, so this should be a `buildPythonModule`.

Reply:
couldn't build with `buildPythonModule` , did `buildPythonPackage` as other modules are also using same
also found [this](https://github.com/NixOS/nixpkgs/issues/1466#issuecomment-31698713) .

Reply:
Yeah sorry I'm an idiot I meant `buildPythonPackage`